### DISCUSSION TITLE
vc-overlay-html 标签使用错误

### DISCUSSION THREAD

**Discussion by hanzhichao110** [zouyaoji/vue-cesium#546] [Q&A]
![image](https://github.com/zouyaoji/vue-cesium/assets/51806965/1c88f89c-9450-44a0-960b-b775b20b66f7)
当我与vc-primitive-tileset 组件一起使用时 会出现 
![image](https://github.com/zouyaoji/vue-cesium/assets/51806965/7e99dc00-5899-45e8-8b55-9a3937416a8e)
![image](https://github.com/zouyaoji/vue-cesium/assets/51806965/e50c083e-261d-49d6-bd99-9557f4fd8693)
这是怎么回事

**Comment by zouyaoji** [comment]
你好，请确保vue-cesium的组件都是作为vc-viewer子组件加载的。

如果是请提供复现的例子。
